auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -618,6 +618,8 @@ export default defineComponent({
           }
           case NrDisplayStates.REJECTED:
           case NrDisplayStates.CONSUMED:
```

Comment:
I'd recommend making a component out of the actions

Reply:
Agreed. I think I made that comment last time but we didn't want to blow up the scope.

It's just because there's a lot of code in here related to the actions, so it might as well be parcelled off and self-contained in a new sub-component.